packages/apps/public/locales/it/translation.json

```
@@ -200,11 +251,15 @@
   "Display overview information for the selected validator, including blocks produced.": "Visualizza riepilogo di informazioni relative al validator selezionato, inclusi i blocchi minati",
   "Distinct stash and controller accounts are recommended to ensure fund security. You will be allowed to make the transaction, but take care to not tie up all funds, only use a portion of the available funds during this period.": "Sono raccomandati differenti account stash e controllers per migliorare la sicurezza sui tuoi fondi. Ti sarà concesso fare transazioni, ma in questa fase non usare tutti i tuoi fondi, ma solo una quota parte di essi",
   "Do not include a tip for the block author": "Non includere una mancia per il minatore del blocco",
+  "Do not transfer any funds directly to a specific account that is associated with a loan or a team. Use the \"Contribute\" action to record the contribution on-chain using the crowdloan runtime module. When the fund is dissolved, after either the parachain lease expires or the loan ending without winning, the full value will be returned to your account by the runtime. Funds sent directly to an account, without using the crowdloan functionality, may not be returned by the receiving account.": "Non trasferire fondi direttamente ad un account specifico associato ad un asta o ad un team. Usa il modulo \"Contribute\" per registrare un contributo on-chain usando la funzione di Crowdloan. Quando il finanziamento è estinto, dopo la scadenza del tempo di leasing della parachain o un asta senza aggiudicazione, il saldo ti sarà restituito automaticamente. I fondi inviati direttamente senza l'utilizzo della funzione di Crowdloan non saranno restituiti" 
```

```suggestion
  "Do not transfer any funds directly to a specific account that is associated with a loan or a team. Use the \"Contribute\" action to record the contribution on-chain using the crowdloan runtime module. When the fund is dissolved, after either the parachain lease expires or the loan ending without winning, the full value will be returned to your account by the runtime. Funds sent directly to an account, without using the crowdloan functionality, may not be returned by the receiving account.": "Non trasferire fondi direttamente ad un account specifico associato ad un asta o ad un team. Usa il modulo \"Contribute\" per registrare un contributo on-chain usando la funzione di Crowdloan. Quando il finanziamento è estinto, dopo la scadenza del tempo di leasing della parachain o un asta senza aggiudicazione, il saldo ti sarà restituito automaticamente. I fondi inviati direttamente senza l'utilizzo della funzione di Crowdloan non saranno restituiti",
```

Missing comma that makes the build fail.